code-challenges.js

```diff
@@ -7,43 +7,112 @@
 // --------------------INSTRUCTOR EXAMPLE: Create a function that takes in any two strings and returns the string with more characters. Use the two sets of test variables provided.
 
 // Set one:
-const fruit1 = "apple"
-const fruit2 = "banana"
+// const fruit1 = "apple"
+// const fruit2 = "banana"
 
-// Set two:
-const fruit3 = "cherry"
-const fruit4 = "kiwi"
+// // Set two:
+// const fruit3 = "cherry"
+// const fruit4 = "kiwi"
 
 
 // --------------------1) Create a function that takes in any given number and determines if the number is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit. Use the test variables provided below. Expected output: "42 is below boiling point", "350 is above boiling point", "212 is at boiling point"
 
-const temp1 = 42
-const temp2 = 350
-const temp3 = 212
+//This function takes any given number and determines if it is below, at, or above boiling point by using greater than/less than/equal to comparisons.
+//The boiling point is 212 degrees Fahrenheit. 
+
+//The input will be temperatures assigned to variables labeled temp1, temp2, and temp3.
+ const temp1 = 42
+ const temp2 = 350
+ const temp3 = 212
+
+const givenNumber = (temperature) => {
+    if (temperature < 212) {
+         return `${temperature} is below boiling point`
+     } else if (temperature === 212) {
+        return `${temperature} is at boiling point`
+    } else if (temperature > 212) {
+        return `${temperature} is above boiling point`
+   }
+ }
```

Comment:
Excellent work on this!!  I always recommend adding an else statement to catch any edge cases that may not fall into this.  For instance what would happen if you passed a string into the function?